### ISSUE TITLE
IOException on startup

### ISSUE THREAD

**Issue by huwpascoe** [OpenRA/OpenRA#11665]
Running on windows

```
System.IO.IOException was unhandled
  HResult=-2147024864
  Message=The process cannot access the file '...\OpenRA\Content\ra\scores.mix' because it is being used by another process.
  Source=mscorlib
  StackTrace:
       at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
       at System.IO.File.InternalDelete(String path, Boolean checkHost)
       at System.IO.File.Delete(String path)
       at OpenRA.Migrations.Run() in ...\OpenRA\OpenRA.Game\Migrations.cs:line 53
       at OpenRA.Game.InitializeMod(String mod, Arguments args) in ...\OpenRA\OpenRA.Game\Game.cs:line 350
       at OpenRA.Game.Initialize(Arguments args) in ...\OpenRA\OpenRA.Game\Game.cs:line 292
       at OpenRA.Program.Run(String[] args) in ...\OpenRA\OpenRA.Game\Support\Program.cs:line 117
       at OpenRA.Program.Main(String[] args) in ...\OpenRA\OpenRA.Game\Support\Program.cs:line 34
```

**Comment by pchote** [comment]
This will be fixed by #11545 after I rework it to remove the migrations code.

**Comment by pchote** [comment]
This was fixed by #11545.
